hack/gen-gen-go.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# A work-around for generating gen.go for grpc packages.
+# This should definitely go away when we came up with a better solution.
```

Comment:
This solution works great. It is much better than many of the methods that employ complex path munging. The key is to limit the scope of what can be done.

The problem is usually that import path setup is done incorrectly, making for many additions in the scripts to deal it. Keeping a script out of the loop will vector you towards doing the correct thing.